parameter/ParameterMgr.cpp

```diff
@@ -615,10 +597,7 @@ bool CParameterMgr::loadSettingsFromConfigFile(string& strError)
     CConfigurableDomains* pConfigurableDomains = getConfigurableDomains();
 
     // Get Xml configuration domains file name
-    string strXmlConfigurationDomainsFilePath = pConfigurableDomainsFileLocation->getFilePath(_strXmlConfigurationFolderPath);
-
-    // Get Xml configuration domains folder
-    string strXmlConfigurationDomainsFolder = pConfigurableDomainsFileLocation->getFolderPath(_strXmlConfigurationFolderPath);
+    string configurationDomainsUri = CXmlDocSource::mkURI(_strXmlConfigurationUri, pConfigurableDomainsFileLocation->getUri());
```

Comment:
"URI" or "Uri" ? Please choose one and stick to it. (I'd choose "Uri" because even though it's an acronym, we have chose lowerCamelCase style)